js/src/market.ts

```diff
@@ -129,8 +138,13 @@ export class Market {
     this._orderbookAddress = orderbookAddress;
     this._baseSplTokenMultiplier = new BN(10).pow(new BN(baseDecimals));
     this._quoteSplTokenMultiplier = new BN(10).pow(new BN(quoteDecimals));
-    this._tickSize = orderbookState.tickSize.toNumber();
-    this._minOrderSize = marketState.minBaseOrderSize.toNumber();
+    this._tickSize = roundUiAmount(
```

Comment:
Changed tickSize and minOrderSize to uiAmounts for backwards compatibility with v3. 

Adding 'BN' options.